cmd/load.go

```diff
@@ -65,6 +65,7 @@ func createLoadCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&params.accountServerURL, "url", "", "", "URL of the account server")
 	cmd.Flags().StringVarP(&params.accountSeed, "seed", "", "", "Seed of the account used to create users")
 	cmd.Flags().StringVarP(&params.username, "user", "", "default", "Default username")
+	cmd.Flags().StringVarP(&params.operatorName, "operator", "", "", "Name used for the operator context")
```

Comment:
how is this handled - after all we are just going to use the operator name provided by the JWT we download?